CMakeLists.txt

```diff
@@ -1147,81 +1159,100 @@ IF(PCRE2_HEAP_MATCH_RECURSE)
 ENDIF(PCRE2_HEAP_MATCH_RECURSE)
 
 IF(PCRE2_SHOW_REPORT)
-  STRING(TOUPPER "${CMAKE_BUILD_TYPE}" buildtype)
-  IF (CMAKE_C_FLAGS)
-    SET(cfsp " ")
-  ENDIF(CMAKE_C_FLAGS)
```

Comment:
I think it is easier to have this set only once, no?

if it was moved to avoid polluting the rest of the script the maybe just unset it after we are done?, not sure how cmake defines scope.

Reply:
Done.